packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -0,0 +1,5 @@
+const CorberWebpackPlugin = require('corber-webpack-plugin');
+
+module.exports = {
+  plugins: [new CorberWebpackPlugin()],
+};
```

Comment:
@trayldev instead of adding this file (which I don't believe is used any more by `react-scripts`) could you add the below line above https://github.com/cactus-denver/Nod-create-react-app/blob/master/packages/react-scripts/config/webpack.config.js#L530 ? I think removing this file and adding that line would then hook Corber and Cordova into the Webpack build system, which might fix the issues we were seeing with `window.cordova` not working ...

```js
      isEnvDevelopment && new CorberWebpackPlugin(),
```

Reply:
This has been completed.